### DISCUSSION TITLE
Welcome to filament-settings-hub Discussions!

### DISCUSSION THREAD

**Discussion by fadymondy** [tomatophp/filament-settings-hub#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bhattji** [comment]
How to add new setting , let say site_theme ="my theme"

**Reply by fadymondy** [reply]
check this 

https://github.com/spatie/laravel-settings
